core/domain/html_validation_service.py

```diff
@@ -1116,6 +1116,11 @@ def add_math_content_to_math_rte_components(html_string):
         html_string.encode(encoding='utf-8'), 'html.parser')
     for math_tag in soup.findAll(name='oppia-noninteractive-math'):
         if math_tag.has_attr('raw_latex-with-value'):
+            # There have been cases where the attr value is empty.
```

Comment:
Suggest adding a link or issue number for context in case the reader wants to better understand such cases.

Reply:
No issue number -- there is only a single case of this on the prod server. It was dealt with manually in an earlier migration (v34) but we can't really do this for snapshots.

Reply:
Can you explicitly call that out in the comment? My concern is as a passerby reading "There have been cases" seems very mysterious and vague, and doesn't really add much justification for why the if check is needed & why it needs to stay beyond "I might break this if I removed it." which isn't so great. Short of a link or issue, the 'v34' context helps provide a cross-reference for which the reader can better understand the need for this. Ideally this would also include a brief explanation of the root cause/why it can occur, but if we don't know that then an explanation for why it's needed now/the tie-in to snapshots/v34 migration should suffice.

Reply:
Done. Thanks for calling that out -- I realize the original comment was quite vague! I've added more clarification.

Also, for info, this whole function will get removed entirely once we migrate everything to the latest states schema version, since it deals with a condition that would no longer exist at that point.